ammo-be/src/shared/typings/Bullet.ts

```diff
@@ -0,0 +1,22 @@
+export type Bullet = {
+    url: string;
+    method:
+        | 'GET'
+        | 'PUT'
+        | 'POST'
+        | 'DELETE'
+        | 'HEAD'
+        | 'CONNECT'
+        | 'OPTIONS'
+        | 'TRACE'
+        | 'PATCH';
+    request: {
+        headers: Record<string, string | string[]>;
+        body: unknown;
+    };
+    response: {
+        headers: Record<string, string | string[]>;
+        body: unknown;
+        status: number;
+    };
+};
```

Comment:
This will clash with the <Bullet /> component on the FE side. We have to agree on a convention, whatever it may be, that will help reduce overlaps (IBullet? TBullet? Or rename the component?)